my-videos/src/components/MyVideos/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { forEach } from 'lodash';
+import './MyVideos.css';
+
+export default function MyVideos({ projectsUrl }) {
+  const [isLoaded, setIsLoaded] = useState(false);
+  const map = {
+    'square' : 'http://placekitten.com/g/100/100',
+    'landscape' : 'http://placekitten.com/g/250/150',
+    'vertical' : 'http://placekitten.com/g/100/200',
+  }
+
+  useEffect(async () => {
+    await fetch(projectsUrl)
+      .then(response => response.json())
+      .then(response => {
+        fetchedData = response.payload.projects;
+        setIsLoaded(true);
+      });
+
+    return fetchedData;
+  }, [isLoaded]);
+
+  return (
+    <div className="wrap">
+      {!isLoaded && (
+        <p>Please hold, while we load your videos...</p>
+      )}
+      {isLoaded && forEach(fetchedData, (project) => {
+        const {
+          aspect_ratio : aspectRatio,
+          presentation,
+          okey,
+          metadata,
+          links
+        } = project;
+
+        const thumbnailUrl = links.cover_image || map[aspectRatio]
+
+        if(aspectRatio == 'square') {
+          return (
+            <div className="thumbnail square">
+              <div className="cover-image">
+                <img src={thumbnailUrl} />
+              </div>
+              <p className="title">{presentation.title}</p>
+              <p className="description">{presentation.description}</p>
+              <span className="updated">{metadata.updated_at}</span>
+            </div>
+          );
+        }
+
+        if(aspectRatio == 'landscape') {
+          return (
+            <div className="thumbnail landscape">
+              <div className="cover-image">
+                <img src={thumbnailUrl} />
+              </div>
+              <p className="title">{presentation.title}</p>
+              <p className="description">{presentation.description}</p>
+              <span className="updated">{metadata.updated_at}</span>
+            </div>
+          );
+        }
+
+        if(aspectRatio == 'verticle') {
```

Comment:
nit (I like to use the word 'nit' to show that I'm noticing something small, but worth pointing out): looks like we've got a typo! 
```suggestion
        if(aspectRatio == 'vertical') {
```